src/hooks/useIcebreakers.ts

```
@@ -0,0 +1,146 @@
+    shared_interests: string[];
+    conversation_prompt: string;
+    profiles?: {
+        full_name: string;
+        avatar_url: string | null;
+        major: string | null;
+    };
+    has_waved: boolean;
+}
+
+interface UseIcebreakersReturn {
+    matches: IcebreakerMatch[];
```

ESLint

[error] Delete `··`

(prettier/prettier)